src/lavinmq/http/controller/queues.cr

```diff
@@ -41,6 +42,21 @@ module LavinMQ
           end
         end
 
+        get "/api/queues/:vhost/:name/unacked" do |context, params|
+          with_vhost(context, params) do |vhost|
+            refuse_unless_management(context, user(context), vhost)
+            unacked_messages = Hash(UInt64, UnackedMessage).new
+            @amqp_server.vhosts[vhost].queues[params["name"]].@consumers.each do |c|
+              c.@channel.@unacked.reject { |u| u.queue.name != params["name"] }.each do |u|
+                if consumer = u.consumer
+                  unacked_messages[u.tag] = UnackedMessage.new(u.tag, consumer.tag, u.delivered_at.to_s)
+                end
+              end
+            end
+            page(context, unacked_messages.each_value)
```

Comment:
if we only use the values (`each_value`), why was a hash created to begin with and not an array?

Reply:
Oops, remnant of something I tried earlier. Changed it.